spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -2269,12 +2269,21 @@ It is registered under the name of `binders` and can be enabled or disabled by s
 [[spring-cloud-stream-overview-metrics-emitter]]
 == Metrics Emitter
 
-Spring Cloud Stream provides a module called `spring-cloud-stream-metrics` that can be used to emit any available metric from the https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-metrics.html[Spring Boot metrics endpoint] to a named channel.
-This module lets operators collect metrics from stream applications without relying on polling their endpoints.
+Spring Boot Actuator provides dependency management and auto-configuration for https://micrometer.io/[Micrometer], an application metrics
+facade that supports numerous https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/reference/htmlsingle/#production-ready-metrics[monitoring systems].
 
-The module is activated when you set the destination name for metrics binding -- for example, `spring.cloud.stream.bindings.applicationMetrics.destination=<DESTINATION_NAME>`.
-`applicationMetrics` can be configured in a similar fashion to any other producer binding.
-The default `contentType` setting of `applicationMetrics` is `application/json`.
+Spring Cloud Stream provides support for emitting any available micrometer-based metrics to a binding destination allowing for periodic
+collection of metric data from stream applications without relying on polling individual endpoints.
+
+Metrics Emitter is activated by simply defining `spring.cloud.stream.bindings.applicationMetrics.destination` property
```

Comment:
Please use this (including the comma at the end): 

Metrics Emitter is activated by defining the `spring.cloud.stream.bindings.applicationMetrics.destination` property,